developer/src/kmc-generate/src/template/kmn-keyboard/source/keyboard.kmn

```diff
@@ -0,0 +1,14 @@
+﻿c keyboard generated from template at $DATE
+c with name "$NAME"
+store(&NAME) '$NAME'
+store(&COPYRIGHT) '$COPYRIGHT'
+store(&VERSION) '10.0'
+store(&KEYBOARDVERSION) '$VERSION'
+store(&TARGETS) '$TARGETS'
+$Icon:store(&BITMAP) '$ID.ico'
+$KVKS:store(&VISUALKEYBOARD) '$ID.kvks'
+$TouchLayout:store(&LAYOUTFILE) '$ID.keyman-touch-layout'
```

Comment:
Do we need an option if the generated keyboard is RTL?
`store(&KMW_RTL)`

Reply:
There are many options that we are not covering in the generator, this is one of them. For now, I think we leave it alone -- otherwise there will be too many parameters to consider.